go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/conduitio/conduit-connector-connectorname
+module github.com/github.com/conduitio-labs/conduit-connector-weather
```

Comment:
Should be `github.com/conduitio-labs/conduit-connector-weather`, no?

Reply:
yes, double github is fixed